boards/arm/efm32gg_stk3701a/doc/index.rst

```diff
@@ -0,0 +1,203 @@
+EFM32GG Series 1 family built on an ARM® Cortex®-M4F processor with excellent
+low power capabilities.
+
+.. figure:: ./efm32gg_stk3701a.jpg
+   :width: 375px
+   :align: center
+   :alt: EFM32GG-SLSTK3701A
+
+   EFM32GG-SLSTK3701A (image courtesy of Silicon Labs)
+
+Hardware
+********
+
+- Advanced Energy Monitoring provides real-time information about the energy
+  consumption of an application or prototype design.
+- Ultra low power 128x128 pixel color Memory-LCD
+- 2 user buttons, 2 LEDs and a touch slider
+- Relative humidity, magnetic Hall Effect and inductive-capacitive metal sensor
+- USB interface for Host/Device/OTG
+- 32 Mb Quad-SPI Flash memory
+- SD card slot
+- RJ-45 Ethernet jack
+- 2 digital microphones
+- On-board Segger J-Link USB debugger
+
+For more information about the EFM32GG11 SoC and EFM32GG-STK3701A board:
+
+- `EFM32GG Series 1 Website`_
+- `EFM32GG11 Datasheet`_
+- `EFM32GG11 Reference Manual`_
+- `EFM32GG-STK3701A Website`_
+- `EFM32GG-STK3701A User Guide`_
+- `EFM32GG-STK3701A Schematics`_
+
+Supported Features
+==================
+
+The efm32gg_stk3701a board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| COUNTER   | on-chip    | rtcc                                |
++-----------+------------+-------------------------------------+
+| ETHERNET  | on-chip    | ethernet                            |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash memory                        |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c port-polling                    |
++-----------+------------+-------------------------------------+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/efm32gg_stk3701a/efm32gg_stk3701a_defconfig``
+
+Other hardware features are currently not supported by the port.
+
+Connections and IOs
+===================
+
+The EFM32GG11 SoC has nine GPIO controllers (PORTA to PORTI), all of which are
+currently enabled for the EFM32GG-STK3701A board.
+
+In the following table, the column **Name** contains pin names. For example, PE2
```

Comment:
Interesting that you picked an example "PE2" that's not an entry in the table.

Reply:
True, I'm afraid this is a case of 'copy/paste, no time to waste'. Modified to PE1, which *is* actually used on the board.